js/form.js

```diff
@@ -0,0 +1,23 @@
+import {pricesPerNight} from './data.js';
+
+const adElementType = document.querySelector('#type');
+const adElementPrice = document.querySelector('#price');
+const adElementTimeIn = document.querySelector('#timein');
+const adElementTimeOut = document.querySelector('#timeout');
+
+const typeChangeHandler = function () {
+  adElementPrice.min = pricesPerNight[adElementType.value];
+  adElementPrice.placeholder = pricesPerNight[adElementType.value];
+};
+
+const onTimeInChange = function () {
+  adElementTimeOut.value = adElementTimeIn.value;
+};
+
+const onTimeOutChange = function () {
+  adElementTimeIn.value = adElementTimeOut.value;
+}
+
+adElementType.addEventListener('change', typeChangeHandler);
```

Comment:
https://up.htmlacademy.ru/javascript/22/criteries#d4
> (на каком элементе) + что случилось + Handler:

```suggestion
adElementType.addEventListener('change', adElementTypeChangeHandler);
```

Reply:
Предлагают убрать `ad` из элемента, а то начинает путаться с `add` от ивент лисенера. Либо, расписать его полностью, как `adverb` хотя бы.

Reply:
поправь остальные тоже